watchtower/__init__.py

```diff
@@ -49,6 +49,59 @@ class WatchtowerWarning(UserWarning):
     pass
 
 
+class CloudWatchFormatter(logging.Formatter):
+    """
+    Log formatter for CloudWatch message. Transforms the logged record message into a compatible message for CloudWatch.
+    This is the default formatter for CloudWatchLogHandler
+
+    :param json_serialize_default:
+        The 'default' function to use when serializing dictionaries as JSON. Refer to the Python standard library
+        documentation on 'json' for more explanation about the 'default' parameter.
+        https://docs.python.org/3/library/json.html#json.dump
+        https://docs.python.org/2/library/json.html#json.dump
+    :type json_serialize_default: Function
+    """
+
+    def __init__(self, *args, json_serialize_default=None, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.json_serialize_default = json_serialize_default or _json_serialize_default
+
+    def format(self, message):
+        if isinstance(message.msg, Mapping):
+            message.msg = json.dumps(message.msg, default=self.json_serialize_default)
+
+        return super().format(message)
+
+
+class CloudWatchJSONFormatter(CloudWatchFormatter):
+    """
+    JSON log formatter for CloudWatch. Transforms the logged record message into a JSON formatted message.
+
+    :param json_serialize_default:
+        The 'default' function to use when serializing dictionaries as JSON. Refer to the Python standard library
+        documentation on 'json' for more explanation about the 'default' parameter.
+        https://docs.python.org/3/library/json.html#json.dump
+        https://docs.python.org/2/library/json.html#json.dump
+    :type json_serialize_default: Function
+    :param fields: A list of fields of the record to include in the CloudWatch Log json object. Defaults to '__all__'.
+    :type fields: list
+    """
+    def __init__(self, *args, fields='__all__', **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields = fields
+
+    def format(self, message):
+        if self.fields == '__all__':
```

Comment:
Please remove the magic value "__all__" and use None as the placeholder for keeping all field values instead.